tools/perf/ResultsComparer/README.md

```diff
@@ -18,33 +32,190 @@ It can be used to compare:
+- `Allocations`
+- `Size`
+- `Bytes` (same as `Size`)
+
 
-## Sample results
+#### Sample Results
 
-| Slower                                                          | diff/base | Base Median (ns) | Diff Median (ns) | Modality|
-| --------------------------------------------------------------- | ---------:| ----------------:| ----------------:| -------:|
-| PerfLabTests.BlockCopyPerf.CallBlockCopy(numElements: 100)      |      1.60 |             9.22 |            14.76 |         |
-| System.Tests.Perf_String.Trim_CharArr(s: "Test", c: [' ', ' ']) |      1.41 |             6.18 |             8.72 |         |
+summary:
+better: 1, geomean: 1.087
+worse: 2, geomean: 1.014
+new (results in the diff that are not in the base): 2
+missing (results in the base that are not in the diff): 1
+total diff: 6
 
-| Faster                              | base/diff | Base Median (ns) | Diff Median (ns) | Modality|
-| ----------------------------------- | ---------:| ----------------:| ----------------:| -------:|
-| System.Tests.Perf_Array.ArrayCopy3D |      1.31 |           372.71 |           284.73 |         |
+| Worse        |          diff/base | Base Allocations | Diff Allocations | Modality|
+| ------------ | ------------------:| ----------------:| ----------------:| --------:|
+| System.Int32 | 1.0866205845157042 |        115446.00 |        125446.00 |         |
+
+| Better                            |          base/diff | Base Allocations | Diff Allocations | Modality|
+| --------------------------------- | ------------------:| ----------------:| ----------------:| --------:|
+| System.String                     | 1.0238671079429735 |        128696.00 |        125696.00 |         |
+| System.Collections.Generic.List<> | 1.0039367088607596 |        237933.00 |        237000.00 |         |
+
+| New                               | diff/base | Base Allocations | Diff Allocations | Modality|
+| --------------------------------- | --------- | ----------------:| ----------------:| -------- |
+| Microsoft.OData.ODataProperty     | N/A       |                  |         45000.00 | N/A     |
+| System.SZGenericArrayEnumerator<> | N/A       |                  |         50147.00 | N/A     |
+
+| Missing                                     | diff/base | Base Allocations | Diff Allocations | Modality|
+| ------------------------------------------- | --------- | ----------------:| ----------------:| -------- |
+| System.Collections.ObjectModel.Collection<> | N/A       |        125001.00 |                  | N/A     |
+
+
+
+### .NET Object Allocation Tracker - Function Allocations
+
+This is the **Functions** tab of the profiler. This report show the number and size of allocations for each method. The methods are grouped by namesace, make sure to expand the namespaces you are interested in, otherwise their methods will not be included in the report when you copy-paste it.
```

Comment:
```suggestion
This is the **Functions** tab of the profiler. This report show the number and size of allocations for each method. The methods are grouped by namespace, make sure to expand the namespaces you are interested in, otherwise their methods will not be included in the report when you copy-paste it.
```